### PR TITLE
MES-4715 without Analytics

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Description
- If a DE starts a test 5 minutes before the test is due to start, a modal will pop up prompting the user to confirm that this was done on purpose.
- Unit tests not implemented.
- Analytics not implemented.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2020-01-30 at 10 12 14

### CODE REVIEW

File: src/components/test-slot/test-outcome/test-outcome.ts [hunk 1]
```diff
@@ -223,9 +251,19 @@ export class TestOutcomeComponent implements OnInit {
...
+    if (this.shouldDisplayCheckStartModal()) {
+      this.displayCheckStartModal();
+      return;
+    }
     this.startOrResumeTestDependingOnStatus();
   }
 
+  shouldDisplayCheckStartModal(): boolean {
+    const retVal = new DateTime().compareDuration(this.slotDetail.start, Duration.MINUTE) > 5;
+    console.log(retVal);
```
Comment (TomBillingtonUK) [inline src/components/test-slot/test-outcome/test-outcome.ts]: console log

**Review by TomBillingtonUK — APPROVED** [review]

File: src/components/test-slot/test-outcome/test-outcome.ts (hunk 1 above)
Comment (tomcrawleyy) [reply src/components/test-slot/test-outcome/test-outcome.ts]: removed

File: src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+journal-early-start-modal {
+  @extend .modal-alert-wrapper;
+
+  .modal-alert {
+    width: 500px;
```
Comment (szabikr) [inline src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss]: where did you get this number from? :)

File: src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  .modal-alert-message {
+    margin-bottom: 25px;
```
Comment (szabikr) [inline src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss]: 24px would be a better representation for our grid

File: src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss (hunk 2 above)
Comment (tomcrawleyy) [reply src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss]: Honestly, its just a number which I had concocted myself as it didn't seem standardised when looking through the other modals, so went with something that made the text appear nicely wrapped - is there a specific value it should be set to?

File: src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss (hunk 3 above)
Comment (tomcrawleyy) [reply src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss]: pushing this change now

File: src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss (hunk 2 above)
Comment (szabikr) [reply src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss]: There must be a specific value, I'm not sure what that is though. Oli or Nad will be able to tell you

**Review by szabikr — APPROVED** [review]
Try to find out what is the actual number of pixels for the modal width. Although I do understand that this is a critical change that has to go in ASAP so if you're running out of time then don't worry

**Review by rossfauldsbjss — APPROVED** [review]
looks good, assuming Szabis comments are adhered to.
One thing - when cherry picking into dev the tests will need to be 
changed to be compatible with ngBullet

**Comment by szabikr** [comment]
ahh the ngBullet is a good shout! Maybe worth adding to the story

File: src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss (hunk 2 above)
Comment (tomcrawleyy) [reply src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss]: We have reached out to Oli, updated this value to 450px, changed the Cancel button text to 'Return to journal' and pushed the changes back, screenshot has been updated to reflect the design change.

**Comment by tomcrawleyy** [comment]
> ahh the ngBullet is a good shout! Maybe worth adding to the story

**Pull request merged**